openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/CacheConfigTests.java

```diff
@@ -39,37 +39,39 @@ public void setup() throws IOException {
         this.tempDir = Files.createTempDirectory("bal-to-openapi-test-out-" + System.nanoTime());
     }
 
-    @Test(description = "Generate OpenAPI spec for service configuration annotation in resource without fields")
+    @Test(description = "Generate OpenAPI spec for service configuration annotation in resource without fields",
```

Comment:
Is there any specific reason for disabling these tests?

Reply:
The tests kept failing without any significant changes to the codebase. The test were disabled expecting this to be an urgent dependency for the Java 21 migration of the Ballerina distribution. Since the changes were not urgently needed for the migration, I will convert this PR to draft so we can sort out the failing tests before merging.

Reply:
Fixed the failing tests in 58cd5396baf78ef2c8e14d233adf857d1667602c. @lnash94 Please check if everything is in order.